modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_8_0/UpgradeDDMStructure.java

```diff
@@ -179,19 +177,15 @@ private void _upgradeDDMStructureLayout() throws Exception {
 
 			try (ResultSet rs = ps1.executeQuery()) {
 				while (rs.next()) {
-					String ddmStructureLayoutDefinition = rs.getString(
-						"structureLayoutDefinition");
-
-					String ddmStructureVersionDefinition = rs.getString(
-						"structureVersionDefinition");
+					String structureLayoutDefinition = rs.getString(
```

Comment:
@marcosapmf, now we can name this variable only as `definition` to keep consistency.